core/src/main/java/de/danoeh/antennapod/core/util/comparator/InReverseChronologicalOrder.java

```diff
@@ -0,0 +1,19 @@
+package de.danoeh.antennapod.core.util.comparator;
+
+import java.util.Comparator;
+
+import de.danoeh.antennapod.core.feed.FeedItem;
+import de.danoeh.antennapod.core.feed.SearchResult;
+
+public class InReverseChronologicalOrder implements Comparator<SearchResult> {
+    /**
+     * Compare items and sort it on chronological order.
+     */
+    @Override
+    public int compare(SearchResult o1, SearchResult o2) {
+        if ((o1.getComponent() instanceof FeedItem) &&
+           (o2.getComponent() instanceof FeedItem))
+            return ((FeedItem) o2.getComponent()).getPubDate().compareTo(((FeedItem) o1.getComponent()).getPubDate());
```

Comment:
Talking about style, IMO extracting either `getComponent()` or `getPubDate()` to variables would make this line easier to read.